scripts/update_repos_version.py

```diff
@@ -1,152 +1,150 @@
+                            if '{}=='.format(module_name) in line:
+                                print '{}=={}'.format(module_name, new_version)
                             else:
                                 print line,
 
+        if not files_changed:
+            # Module name wasn't found in the requirements files.
+            logging.error("Module name '{}' not found in repo {} - skipping.".format(module_name, repo_url))
+            continue
+
         # Add/commit the files.
-        ret_code = subprocess.call(['git', 'commit', '-am', 'Updating {} requirement to version {}'.format(RELEASE_UTIL_REPO_NAME, new_version)])
+        ret_code = subprocess.call(['git', 'commit', '-am', 'Updating {} requirement to version {}'.format(module_name, new_version)])
         if ret_code:
-            err_msg = "Failed to add and commit changed files to repo {}".format(repo)
-            logging.error(err_msg)
-            raise Exception(err_msg)
+            logging.error("Failed to add and commit changed files to repo {}".format(repo_url))
+            continue
 
         # Push the branch.
         ret_code = subprocess.call(['git', 'push', '--set-upstream', 'origin', branch_name])
         if ret_code:
-            err_msg = "Failed to push branch {} upstream for repo {}".format(branch_name, repo)
-            logging.error(err_msg)
-            raise Exception(err_msg)
+            logging.error("Failed to push branch {} upstream for repo {}".format(branch_name, repo_url))
+            continue
 
         # Create a PR with an automated message.
+        rollback_branch_push = False
         try:
             response = gh.create_pull(
-                title='Change {} version.'.format(RELEASE_UTIL_REPO_NAME),
-                body='Change the required version of {} to {}.\n\n@edx-ops/pipeline-team Please review and tag appropriate parties.'.format(RELEASE_UTIL_REPO_NAME, new_version),
+                title='Change {} version.'.format(module_name),
+                body='Change the required version of {} to {}.\n\n@edx-ops/pipeline-team Please review and tag appropriate parties.'.format(module_name, new_version),
                 head=branch_name,
                 base='master'
             )
         except UnknownObjectException:
-            logging.error('No GitHub PR-creating permissions - did you set GITHUB_TOKEN or GITHUB_USERNAME/PASSWORD?')
+            logging.error('No GitHub PR-creating permissions - did you set GITHUB_TOKEN?')
+            rollback_branch_push = True
         except:
-            logging.error('Failed to create PR for repo {}!'.format(repo))
+            logging.error('Failed to create PR for repo {}!'.format(repo_url))
+            rollback_branch_push = True
         else:
-            logging.info('Created PR for repo {}: {}'.format(repo, response.html_url))
+            logging.info('Created PR for repo {}: {}'.format(repo_url, response.html_url))
```

Comment:
nit: since the create_pull response returns a PullRequest, it is possible and would be nice to include the PR.id in this logging statement.